examples/send-and-receive-emails/send-and-receive-emails-2-pop.bal

```diff
@@ -0,0 +1,24 @@
+import ballerina/email;
+import ballerina/io;
+
+public function main() {
+    // Create the client with the connection parameters, host, username, and password. An error is received in failure.
+    // Default port number `995` is used over SSL with these configurations.
+    email:PopClient|email:Error popClient = new ("pop.email.com", "reader@email.com", "pass456");
```

Comment:
The column length issue applies here as well. Shall we refactor them

Reply:
Corrected.